packages/commonwealth/client/scripts/app.ts

```diff
@@ -58,39 +58,43 @@ export async function initAppState(
         app.config.nodes.clear();
         app.user.notifications.clear();
         app.user.notifications.clearSubscriptions();
-        data.nodes
+        data.result.nodes
           .sort((a, b) => a.id - b.id)
           .map((node) => {
             return app.config.nodes.add(NodeInfo.fromJSON(node));
           });
-        data.chains
-          .filter((chain) => chain.active)
+        data.result.chainsWithSnapshots
```

Comment:
I don't understand why this is named `chainsWithSnapshots` -- what does it mean?

Reply:
We used to return the snapshot data as part of chains- in order to keep things working i separate it out so we get an object like `{
    chain: ChainInstance;
    snapshot: string[] | null;
  }`